crypto/keys/ed25519/ed25519.go

```diff
@@ -156,6 +156,7 @@ func (pubKey *PubKey) Address() crypto.Address {
 		panic("pubkey is incorrect size")
 	}
 	return crypto.Address(tmhash.SumTruncated(pubKey.Key))
+	// TODO use ADR-28: return address.Hash(proto.MessageName(pubKey), pubKey.Key)
```

Comment:
I'm not sure what this TODO means:

1. Use ADR-028 for ed25519 in tendermint AND sdk? Is TM okay with this big breaking change?
2. Use different ed25519 addresses in tendermint and sdk? What are the implications of this?

Reply:
This is just a note for a code we will have to use when we will decide how to solve the SDK - TM problem I discussed [above](https://github.com/cosmos/cosmos-sdk/pull/8415#issuecomment-774680687).
I can change `TODO` to `NOTE` if you prefer.